### PR TITLE
Task: logapi and minio-client api

### PR DESCRIPTION
**Pull request by ahmedzidan**
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: logsearchapi/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - '5432:5432'
+    volumes:
+      - db:/var/lib/postgresql/data
+
+  api:
+    image: ahmedzidan204/logsearchapi:v1.0.0 #change to 0chain registry
```
Comment (peterlimg) [inline logsearchapi/docker-compose.yaml:17]: Use docker image from 0chaindev

**Review by peterlimg — CHANGES_REQUESTED** [review]
A few changes would be required as show below:
- Code structures, if the clientapi and logsearchapi are two main services, then put them in the `cmd` package, so should be `cmd/clientapi` and `cmd/logsearchapi`. 
- All go code filenames should be in lower case connected with underscore, e.g `create_bucket.go` instead of `createBucket.go`. 
- For `client-api`, instead of putting all API in one `/?action=` endpoint, we need to separate them into different endpoints.

File: logsearchapi/docker-compose.yaml (hunk 1 above)
Comment (Hassaan008) [reply logsearchapi/docker-compose.yaml:17]: @ahmedzidan

Comment (ahmedzidan) [reply logsearchapi/docker-compose.yaml:17]: This file will be deleted, I've already pushed the latest docker-compose file in environment folder

Comment (ahmedzidan) [reply logsearchapi/docker-compose.yaml:17]: Hi @peterlimg thanks for your comments, let me reply  to them here
1- Use cmd for logsearchapi and client API?  these are two different services has nothing to do with the core package of minioserver, please have a look here to minioOperator repo and see where they put the logsearchapi. https://github.com/minio/operator/tree/master/logsearchapi

Comment (ahmedzidan) [reply logsearchapi/docker-compose.yaml:17]: 2-  use snake case? yeap sure we can use that I can change it

**Pull request merged**